File: issuer-list-base/src/main/java/com/adyen/checkout/issuerlist/IssuerListInputData.java
/*
 * Copyright (c) 2019 Adyen N.V.
 *
 * This file is open source and available under the MIT license. See the LICENSE file for more info.
 *
 * Created by caiof on 25/4/2019.
 */

package com.adyen.checkout.issuerlist;

import androidx.annotation.Nullable;

import com.adyen.checkout.base.component.InputData;

public class IssuerListInputData implements InputData {

    private IssuerModel mSelectedIssuer;

    void setSelectedIssuer(@Nullable IssuerModel selectedIssuer) {
        mSelectedIssuer = selectedIssuer;
    }

    @Nullable
    IssuerModel getSelectedIssuer() {
        return mSelectedIssuer;
    }
}

File: issuer-list/src/main/java/com/adyen/checkout/issuerlist/internal/ui/model/IssuerListInputData.kt
/*
 * Copyright (c) 2019 Adyen N.V.
 *
 * This file is open source and available under the MIT license. See the LICENSE file for more info.
 *
 * Created by caiof on 25/4/2019.
 */
package com.adyen.checkout.issuerlist.internal.ui.model

import androidx.annotation.RestrictTo
import com.adyen.checkout.components.core.internal.ui.model.InputData

@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
data class IssuerListInputData(
    var selectedIssuer: IssuerModel? = null,
) : InputData
